github/OrganizationCustomProperty.py

```diff
@@ -0,0 +1,169 @@
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2024 Jacky Lam <jacky.lam@r2studiohk.com>                          #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+from __future__ import annotations
+
+from typing import Any
+
+from github.GithubObject import Attribute, NonCompletableGithubObject, NotSet, Opt, is_optional
+
+
+class CustomProperty:
```

Comment:
Should we rename this class to something less confusing (`CustomProperty`/`OrganizationCustomProperty`)?